### ISSUE TITLE
Set IP in ServiceStatus rather than ServiceSpec

### ISSUE THREAD

**Issue by samcday** [pthomison/tailscale-ingress-controller#1]
Hello! Interesting looking project you have here :+1: I was looking for something just like this.

I think it might make more sense to set the service tailnet LB IP on [.status.loadBalancer.ingress](https://kubernetes.io/docs/reference/kubernetes-api/service-resources/service-v1/#ServiceStatus), rather than on [.spec.externalIPs](https://github.com/pthomison/tailscale-load-balancer-controller/blob/main/controllers/service_controller.go#L141). WDYT?

**Comment by pthomison** [comment]
Hey @samcday! Thanks for the interest!

So TBH I came up with the .spec.externalsIPs solution just by hacking around && seeing what my automation stack (nginx-ingress-controller && external-dns) would pick up on. Based on your input, I updated the controller to inject the IP into .status.LoadBalancer.Ingress.IP && I like it! It let me tighten up the RBAC and I'm injecting the tailscale MagicDNS hostname as well (under .status.LoadBalancer.Ingress.Hostname). Not sure if anything uses that for automation, but figured I might as well when messing around in there haha.

Here's the [commit](https://github.com/pthomison/tailscale-load-balancer-controller/commit/5d5d17166f82ac6eacb05a081d56a156fbf38da7) that added the change. Working on getting the images built then will get a new helm chart version released!

Let me know if you have any other thoughts or ideas! Thanks again
